core/trino-main/src/main/java/io/trino/testing/TestingSession.java

```diff
@@ -59,6 +60,7 @@ public static SessionBuilder testSessionBuilder(SessionPropertyManager sessionPr
                 .setTimeZoneKey(DEFAULT_TIME_ZONE_KEY)
                 .setLocale(ENGLISH)
                 .setRemoteUserAddress("address")
-                .setUserAgent("agent");
+                .setUserAgent("agent")
+                .setSystemProperty(ADAPTIVE_PARTIAL_AGGREGATION_MIN_ROWS, "1000");
```

Comment:
the default value of 100K is way too large for tests (it will never fire), and I think it's good to check the adaptive part outside of unit tests but checking all possible aggregation variation with and without adaptation seems too much.

Reply:
That is a bit fishy. For me it's ok, but I guess someone can have problems with that.

Reply:
I think this is an example of a general issue with trino properties in tests. Some properties like this one, are targeted at larger data scales, which means the functionality behind the property will never fire if not explicitly tested.
This is especially true for memory-related properties e.g. `task.max-partial-aggregation-memory`.
It seems to me, we should use scaled-down property values in "query" tests.

Reply:
Do not change the property here. Change property in unit tests. You can also add another child of `AbstractTestAggregations` with minimal PA limits.

Reply:
> wouldn't it be better if all query tests were run with the lower limit and not just AbstractTestAggregations?

no. We specifically split different queries into different abstract test classes (join, aggregation, etc) so that we don't have to cross test everything. It's a mess.